runtime/parser2/expression.go

```diff
@@ -131,65 +170,47 @@ func setLeftDenotation(tokenType lexer.TokenType, leftDenotation leftDenotationF
 
 func init() {
 
-	define(infix{
+	define(binary{
+		tokenType:        lexer.TokenLess,
+		leftBindingPower: 60,
```

Comment:
Just a thought, would be nice to have the `bindingPower`s defined somewhere + perhaps relative to each other.

Reply:
Something we can do in the future though!

Reply:
Yep, that's a good idea! It would be nice to have something like Swift's precedence group declarations (in Go, not in Cadence 😄):
https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID550,
i.e. declare relative and then calculate actual numbers at the end

Reply:
Yeah thats exactly what I was thinking, similar to a constraint solver in a way.